producers/mimosa26_sbg/include/M26Producer.hh

```diff
@@ -0,0 +1,90 @@
+#ifndef M26PRODUCER_HH
+#define M26PRODUCER_HH
+
+// EUDAQ includes
+#include "eudaq/Producer.hh"
+#include "eudaq/Timer.hh"
+#include "eudaq/Configuration.hh"
+//#include "eudaq/Utils.hh"
+//#include "eudaq/Logger.hh"
+
+// IRC DLL header
+#include "iphc_run_ctrl_exp.h"
```

Comment:
Not to be confused with https://en.wikipedia.org/wiki/Internet_Relay_Chat
;) Meybe another name would be more adequate.